src/en/models-upgrade.md

```diff
@@ -44,6 +44,8 @@ run on each machine Juju creates, including controllers.
 
 Overview:
 
+- A controller admin user can upgrade any model within their controller and a
```

Comment:
I would make these two separate things

 - The admin user of a controller can upgrade any model within that controller
 - The creator of a model can upgrade that model

Reply:
Ok, but I preserved the user terminology used elsewhere. Also, it really is "model owner". A user can be made an owner without having created the model.